util/buildflags/export.go

```diff
@@ -259,9 +260,16 @@ func (w *csvBuilder) Write(key, value string) {
 	if w.sb.Len() > 0 {
 		w.sb.WriteByte(',')
 	}
-	w.sb.WriteString(key)
-	w.sb.WriteByte('=')
-	w.sb.WriteString(value)
+
+	var attr strings.Builder
+	writer := csv.NewWriter(&attr)
```

Comment:
Btw, every call to `csv.NewWriter` does a 4KB memory allocation. So maybe only call it if there is a comma in value?

Reply:
A comma or a quote maybe?

Reply:
Done!